x11-misc/polybar/files/versioned-docdir.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/doc/CMakeLists.txt b/doc/CMakeLists.txt
```

Comment:
Does not follow patch file naming

Reply:
The wiki says there are no fixed rules regarding patch file names. What did you have in mind?

I figured prepending `${P}` would not be needed because this patch is not version specific.

Reply:
I would use latest release version the patch was introduced.